react-native/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../tslint.json",
+    "rules": {
+        // This package uses the Function type for good reason and `object` is TypeScript 2.2 only.
+        "forbidden-types": false
```

Comment:
Regarding `Function`, there’s for instance this profile function that wraps any given function: https://github.com/facebook/react-native/blob/fbd1b0a6654d87b236a382d52575d637f87acf24/Libraries/Performance/Systrace.js#L183-L192